addons/l10n_in/models/template_in.py

```diff
@@ -49,3 +49,10 @@ def _get_in_account_cash_rounding(self):
                 'loss_account_id': 'p213201',
             }
         }
+
+    @template('in', 'account.fiscal.position')
+    def _update_fiscal_position(self):
```

Comment:
No need to have it in the csv if we have it here.

Reply:
@jco-odoo can you clarify? are you referring to this https://github.com/odoo/odoo/pull/147812#discussion_r1453145217 ?

Reply:
This method should already create it when installing the CoA.  (and putting it in the csv does the same)

Reply:
Still we might need this method because company already having state while setting up CoA, sets the intrastate state for the Fiscal

Reply:
So @jco-odoo, are you suggesting that we completely remove the fiscal intrastate and interstate data from csv and we return a complete set of data of intrastate and interstate from here?

That would work thanks for the suggestion :new_moon_with_face: